tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -343,14 +355,14 @@ func deployment(labelSelector string) (name, owner, uid string, err error) {
 	return
 }
 
-func podLogs(labelSelector string, container string) {
-	pod, err := podID(labelSelector)
+func podLogs(labelSelector string, container string, kubeconfig string) {
+	pod, err := podID(labelSelector, localCluster)
```

Comment:
I guess that I don't understand what pod's log are going to be displayed here. Is the remote pod being obtained here? What about the passed in kubeconfig (remote or local). I have to say I'm confused.